README.md

```diff
@@ -120,6 +120,69 @@ import utc from '@momentjs/moment/timezones/utc/'; // Note trailing slash
 // Error: folders cannot be imported (there is no index.* magic)
 ```
 
+### 2. Imports Field
+
+> **To avoid conflict with `node_modules` packages, the current proposal prefixes all imports with `#name`, so that the fact that an alias is being imported is clear. Whether this restriction is maintained in the final proposal, or what symbol is used is still TBD.**
+
+#### Example
+
+For the same example package as provided for `"exports"`, consider if we wanted to make the `timezones` implementation something that is only referenced internally by code within `@momentjs/moment`, instead of exposing it to external importers.
+
+```js
+{
+  "name": "@momentjs/moment",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "./dist/index.js",
+  "imports": {
+    "#timezones/": "./data/timezones/",
+    "#timezones/utc": "./data/timezones/utc/index.mjs",
+    "#external-feature": "external-pkg/feature",
+    "#moment/": "./"
+  }
+}
+```
+
+As with package exports, mappings are mapped relative to the package base, and keys that end in slashes can map to folder roots.
+
+The resolution algorithms remain the same except `"exports"` provide the added feature that they can also map into third-party packages that would be looked up in node_modules, including to subpaths that would be in turn resolved through `"exports"`. There is no risk of circular resolution here, since `"exports"` themselves only ever resolve to direct internal paths and can't in turn map to aliases.
```

Comment:
what if package A has an imports map of “x” to “B/y”, and package B has an imports map of “A/x”?

Reply:
Do you mean something like:

A/main.js `import '#x'`
B/y.js: `import 'A/main.js'`

where `#x -> B/y` and `B/y -> B/y.js` is an exports map in B?

That would construct a cycle between the modules, yes, but the cycle point was about the resolver itself getting caught in a resolution cycle.

For example, if we supported package exports mapping into external packages then you could set up a cycle where `B/x` -> `A/x` -> `B/x` sort of thing, where the resolver cannot complete the resolve operation.